crowdsec-docs/docs/bouncers/cloudflare-workers.mdx

```diff
@@ -97,6 +97,14 @@ You can run `sudo crowdsec-cloudflare-worker-bouncer -g <CLOUDFLARE_TOKEN1>,<CLO
 
 :::
 
+:::note
+By default the generated config only sets up captcha remediation for all the zones.
```

Comment:
we should mention the word "challenge" right? that's the vocabulary they use in CF right?

Reply:
fixed